docs/jwst/straylight/main.rst

```diff
@@ -68,4 +63,14 @@ where,
 
 The radius of influence :math: 'R' and the exponent :math: 'k' are variables that 
 can be adjusted to the actual problem. The default values for these parameters are
-:math:`R = 50` pixels and :math:`k = 1`.
\ No newline at end of file
+:math:`R = 50` pixels and :math:`k = 1`.
+
+
+The second algorithm uses a 'regions' reference file type that is stored in
```

Comment:
We really can't be referring to the regions information as a reference file type. A reference file is a physical file that's stored in and retrieved from CRDS and gets fetched when the step runs. To me it sounds like the regions information may originally come from one of the official CRDS reference files (the distortion ref file, the actual regions ref file, ?) that gets loaded during the execution of the assign_wcs step and assign_wcs is then storing that information as part of the rest of the gwcs object that it creates (and stores in the ASDF extension)? So the info is available later for this step to use. If that's the case, then that's how it should be explained in the docs. So while the straylight step may not fetch and load the regions reference file directly, it is dependent upon having previously run the assign_wcs step to do that loading, so the info is available for the straylight step.